File: Java/step0/context-invoices/src/main/java/org/cqrs101/invoices/commands/CreateInvoice.java
package org.cqrs101.invoices.commands;

import java.util.UUID;
import org.cqrs.Command;

public class CreateInvoice extends Command {

    private UUID id;
    private UUID customerId;

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public UUID getCustomerId() {
        return customerId;
    }

    public void setCustomerId(UUID userId) {
        this.customerId = userId;
    }
}

File: RoadToCqrs/Cqrs04.Test/Domain/Commands/CreateInvoice.cs
using System;

namespace Cqrs04.Test.Domains.Invoices.Commands
{
    public class CreateInvoice
    {
        public CreateInvoice(Guid invoiceId,Guid customerId)
        {
            InvoiceId = invoiceId;
            CustomerId = customerId;
        }
        public Guid InvoiceId { get; set; }
        public Guid CustomerId { get; set; }
    }
}
